### PR TITLE
When user follows more than 5 concepts, limit each one to show only 2…

### PR DESCRIPTION
**Pull request by debugwand**
… stories

### CODE REVIEW

File: functions/create-email/lib/group-content/populate-follows/index.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
 'use strict';
-
 const conceptModel = require('../../../models/concept-model');
 const populateContent = require('./populate-content');
 
 module.exports = function populateFollows (follows, content) {
 
 	const includedContent = [];
-
+	const content_limit = (follows.length > 5) ? 2 : 4;
```
Comment (laurieboyes) [inline functions/create-email/lib/group-content/populate-follows/index.js]: Camel case?

File: functions/create-email/test/lib/group-content/populate-follows/index.spec.js [hunk 2]
```diff
@@ -10,14 +11,30 @@ function getFollows(names) {
 	return followFixtures.filter(f => names.indexOf(f.name) !== -1);
 }
 
-function getContentTitles(content) {
-	return content.map(c => c.title);
-}
-
 describe('#populateFollows', () => {
+	const populateContentStub = sinon.stub().returns({
```
Comment (laurieboyes) [inline functions/create-email/test/lib/group-content/populate-follows/index.spec.js]: 👍  +1 for isolating this module for unit testing

**Comment by laurieboyes** [comment]
dope 👍

**Comment by keirog** [comment]
👍

File: functions/create-email/lib/group-content/populate-follows/index.js (hunk 1 above)
Comment (debugwand) [inline functions/create-email/lib/group-content/populate-follows/index.js]: will fix, sorry

**Pull request merged**